src/app/features/import/import-column-mapping/import-column-mapping.component.ts

```diff
@@ -63,7 +61,7 @@ export class ImportColumnMappingComponent {
   allProps: string[] = [];
 
   /** properties that need further adjustments through a component */
-  mappingCmp: { [key: string]: ComponentType<AbstractValueMappingComponent> };
+  valueMapper: { [key: string]: ImportValueMapping };
```

Comment:
I am not too happy with the name of the property and the interface. 
I think it would be much clearer to call it `dataTyperMap` and the `DefaultDataType` interface.